frontend/src/integrations/linkedin/components/linkedin-settings-drawer.vue

```diff
@@ -0,0 +1,193 @@
+              profiles to start tracking LinkedIn activities
+            </div>
+          </div>
+        </div>
+      </el-form>
+    </template>
+
+    <template #footer>
+      <div
+        class="flex grow items-center"
+        :class="
+          hasFormChanged ? 'justify-between' : 'justify-end'
+        "
+      >
+        <el-button
+          v-if="hasFormChanged"
+          class="btn btn-link btn-link--primary"
+          @click="doReset"
+          ><i class="ri-arrow-go-back-line"></i>
+          <span>Reset changes</span></el-button
+        >
+        <div class="flex gap-4">
+          <el-button
+            class="btn btn--md btn--bordered"
+            @click="isVisible = false"
+          >
+            <app-i18n code="common.cancel"></app-i18n>
+          </el-button>
+          <el-button
+            class="btn btn--md btn--primary"
+            :class="{
+              disabled: !hasFormChanged || loading
+            }"
+            :loading="loading"
+            @click="hasFormChanged ? connect() : undefined"
+          >
+            {{
+              integration.settings?.organizations.length > 0
+                ? 'Update'
+                : 'Connect'
+            }}
+          </el-button>
+        </div>
+      </div>
+    </template>
+  </app-drawer>
+</template>
+<script>
+export default {
+  name: 'AppLinkedInConnectDrawer'
+}
+</script>
+<script setup>
+import {
+  defineEmits,
+  defineProps,
+  computed,
+  ref,
+  watch
+} from 'vue'
+import { CrowdIntegrations } from '@/integrations/integrations-config'
+import { useStore } from 'vuex'
+
+const store = useStore()
+
+const props = defineProps({
+  modelValue: {
+    type: Boolean,
+    default: false
+  },
+  integration: {
+    type: Object,
+    default: () => {}
+  }
+})
+
+const emit = defineEmits(['update:modelValue'])
+const organizations = computed(
+  () => props.integration.settings?.organizations
+)
+const selectedOrg = computed(() =>
+  organizations.value.find((o) => o.inUse === true)
+)
+
+const model = ref(
+  selectedOrg.value ? selectedOrg.value.id : null
+)
+const loading = ref(false)
+
+const logoUrl =
+  CrowdIntegrations.getConfig('linkedin').image
+
+const isVisible = computed({
+  get() {
+    return props.modelValue
+  },
+  set(value) {
+    return emit('update:modelValue', value)
+  }
+})
+
+const hasFormChanged = computed(() => {
+  if (selectedOrg.value === undefined) {
+    return model.value !== null
+  } else {
+    return model.value !== selectedOrg.value.id
+  }
+})
+
+const doReset = () => {
+  model.value = selectedOrg.value
+    ? selectedOrg.value.id
+    : null
+}
+
+const connect = async () => {
+  loading.value = true
+  await store.dispatch(
+    'integration/doLinkedinOnboard',
+    model.value
+  )
+  loading.value = false
+  isVisible.value = false
+}
+
+watch(isVisible, (newValue, oldValue) => {
+  if (newValue) {
+    window.analytics.track('LinkedIn: settings drawer', {
```

Comment:
I think for reddit we are using `Reddit: connect drawer`. Should we use the same here?

Reply:
This is not really a connect drawer, since it only updates the integration settings and invokes integration re-onboard
(For the use-case of a LinkedIn account that administrates a single organization page, the connect-drawer is not even necessary)

Reply:
Sounds good! Just making sure that it wasn't an inconsistency